server/db/seed.js

```diff
@@ -32,16 +32,21 @@ async function seed() {
     conversationId: santaigoConvo.id,
     senderId: santiago.id,
     text: "Where are you from?",
+    readStatus: false,
```

Comment:
You don't need to add these in since the default value is false 🙂

Reply:
noted!